src/components/ReportActionItem/MoneyRequestAction.tsx

```diff
@@ -91,15 +89,7 @@ function MoneyRequestAction({
             return;
         }
 
-        // If the childReportID is not present, we need to create a new thread
```

Comment:
Are we removing this because we're always creating transaction threads in the backend, and `action?.childReportID` will be set to the ID of that transaction thread?

Reply:
Yes. We've discussed it briefly here: https://github.com/Expensify/App/pull/37232#issuecomment-1975285144